registration/src/main/proto/lobby/registration/conference.proto

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright 2015, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package lobby.registration;
+
+option java_package="com.lobby.registration";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+
+// The conference created by a business customer.
+message Conference {
+
+    // The unique conference id.
+    ConferenceId conference_id = 1;
+
+    // The conference locator access code.
+    // Used by a business customer (conference owner) to view conference details.
+    // Generated when the business customer creates the conference.
+    string access_code = 2;
+
+    // The conference name. Used in the slug.
+    string name = 3;
+
+    // The conference description.
+    string description = 4;
+
+    // The part of the URL used to access the conference.
+    // Consists of conferenceId + conferenceName (name is cut).
```

Comment:
OK. I thought we can use the conference name in the slug, but since we have GUID instead of long ID, I think we will form slugs the other way.